apps/dot_ext/views/authorization.py

```diff
@@ -64,6 +80,30 @@ def form_valid(self, form):
 
         self.success_url = uri
         log.debug("Success url for the request: {0}".format(self.success_url))
+
+        # Extract code from url
+        url_query = parse_qs(urlparse(self.success_url).query)
+        code = url_query.get('code', [None])[0]
+
+        # Get auth uuid from request session
+        auth_uuid = self.request.session.get('auth_uuid', None)
+
+        # Create AuthFlowUuid instance to pass auth_uuid for token related logging
+        try:
+            # Remove if already existing
+            auth_flow_uuid = AuthFlowUuid.objects.get(code=code)
+            auth_flow_uuid.remove()
+        except AuthFlowUuid.DoesNotExist:
+            pass
```

Comment:
As we discussed during PR review, the PK of the AuthFlowUuid model should protect it from this case, so I think this check can be removed.

Reply:
I agree. Just removing it now :+1: